### PR TITLE
#166237760 Integrate HoundCI with badge attached to readme

### PR DESCRIPTION
**Pull request by ayodejiAA**
#### What does this PR do?
Integrate HoundCI with badge attached to readme

#### Description of Task to be completed?
- [x] Setup HoundCI 
- [x] Add houndCi badge to readme

#### What are the relevant pivotal tracker stories?
[#166237760](https://www.pivotaltracker.com/story/show/166237760)

### CODE REVIEW

**Review by tolulope-od — APPROVED** [review]
Good stuff Ayo. Thank you 👍

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
+
```
Comment (tolulope-od) [inline .hound.yml]: Hey Ayo, great work here. However, I think you might need to add the `esversion` config, otherwise, Hound will flag all our ES6 code

**Comment by chialuka** [comment]
Looks good to me. Well done Ayo

File: .hound.yml (hunk 1 above)
Comment (ayodejiAA) [reply .hound.yml]: Thanks for the review, @tolulope-od. You are right. Without the esversion, hound-bot will always flag our PR, but the issue will be resolved when we configure eslint.

Comment (tolulope-od) [reply .hound.yml]: Ok, thanks for making that clear.

**Review by chialuka — APPROVED** [review]
Looks good. Now we have Hound

**Review by akpante3 — APPROVED** [review]

**Review by Nerocodes — APPROVED** [review]
Great job.

**Pull request merged**